### DISCUSSION TITLE
Installing beta Frigate integration for HA

### DISCUSSION THREAD

**Discussion by PatSloan** [blakeblackshear/frigate#9301] [Ask A Question]
I am running the rc 1 running in a docker container and have a question. how do you install a beta of the HA frigate integration as required by rc 1?  Currently only have the production release of the Frigate integration installed in HA.

https://github.com/blakeblackshear/frigate-hass-integration/releases/tag/v5.0.0-beta.3

**Comment by NickM-27** [comment]
Go to HACS and enable beta downloads then redownload

**Comment by PatSloan** [comment]
awesome.  Thanks.
